reactivesearch/src/index.js

```diff
@@ -1,7 +1,117 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import  { createRoot } from 'react-dom/client';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+
+//import OpenLogClient from './js/OpenLogClient'
+var OpenLogClient = require('./ts/OpenLogClient.ts').OpenLogClient;
+
+const {
+	CollectorModule,
+	Context,
+	cookieSessionResolver,
+	debounce,
+	DefaultWriter,
+	FiredSearchCollector,
+	InstantSearchQueryCollector,
+	positionResolver,
+	Query,
+	Sentinel,
+	Trail,
+	TrailType,
+	DebugWriter,
+	QueryWriter,
+	TrailWriter,
+	JSONEnvelopeWriter,
+	RedirectCollector,
+	BrowserTrackingWriter,
+	ProductClickCollector,
```

Comment:
Do we need all of these?

Reply:
I'm working through what we need or not.